definitions/procedures/puppet/enable_puppet.rb

```diff
@@ -0,0 +1,32 @@
+module Procedures::Puppet
+  class EnablePuppet < ForemanMaintain::Procedure
+    metadata do
+      description 'Enable Puppet feature'
+    end
+
+    def run
+      feature(:installer).run(installer_arguments_enabling_puppet.join(' '), :interactive => false)
+    end
+
+    private
+
+    # TODO: This works only for a katello scenario, we should consider vanilla foreman
+    def installer_arguments_enabling_puppet
+      options = [
+        '--enable-puppet',
+        '--puppet-server true',
+        '--foreman-proxy-puppet true',
+        '--foreman-proxy-puppetca true',
+        '--foreman-proxy-content-puppet true',
+        '--puppet-server-foreman-ssl-ca /etc/pki/katello/puppet/puppet_client_ca.crt',
+        '--puppet-server-foreman-ssl-cert /etc/pki/katello/puppet/puppet_client.crt',
+        '--puppet-server-foreman-ssl-key /etc/pki/katello/puppet/puppet_client.key'
```

Comment:
Can we hide this behind an if `answers.key?('foreman_proxy_content')` similar to the disable part? That way it works for vanilla Foreman too.